src/message-types.ts

```diff
@@ -0,0 +1,47 @@
+export default class MessageTypes {
+  public static GQL_CONNECTION_INIT = 'connection_init';
+  public static GQL_CONNECTION_ACK = 'connection_ack';
+  public static GQL_CONNECTION_ERROR = 'connection_error';
+  // NOTE: This one here don't follow the standard due to connection optimization
+  public static GQL_CONNECTION_KEEP_ALIVE = 'ka';
+  public static GQL_CONNECTION_TERMINATE = 'connection_terminate';
+  public static GQL_START = 'start';
+  public static GQL_DATA = 'data';
+  public static GQL_ERROR = 'error';
+  public static GQL_COMPLETE = 'complete';
+  public static GQL_END = 'end';
```

Comment:
What is the difference between `COMPLETE` and `END`? If end is to stop an operation, we should call it `STOP`.